daf/Configurator.cpp

```diff
@@ -122,19 +129,19 @@ namespace DAF
                 }
 
                 if (ACE::debug()) { // Allow for debug output from parser
-                    ACE_DEBUG((LM_DEBUG, ACE_TEXT("Line[%03d]\t%s=%s\n"), line_no, cfgKey.c_str(), cfgArg.c_str()));
+                    ACE_DEBUG((LM_DEBUG, ACE_TEXT("Line[%03d] %s=%s\n"), line_no, cfgKey.c_str(), cfgArg.c_str()));
                 }
 
                 it->second.push_back(mapped_type::value_type(cfgKey, cfgArg));
             }
             catch (const char *what_error) {
```

Comment:
If all we are throwing here are the exception strings then perhaps we could just have these contained within an actual InitializationException when thrown initially.